### DISCUSSION TITLE
After a crash I've "lost" some xch (solved)

### DISCUSSION THREAD

**Discussion by mavaro** [Chia-Network/chia-blockchain#11602] [Support]
Hi everyone!

I'm using chia 1.3.5 in my full node server with three harvesters. Last night after a crash due to an electricity shortage I started again my full node and I'd realized that despite being synced I've lost some of the xch I had in the wallet before the crash. The GUI says synced but it reports a lower last height farmed (~1.6K) than the real one I know (~1.9K) so I think it's not really in sync despite what it says...

Anyone can help me with this?
Thank you very much in advance.

**Comment by mavaro** [comment]
Solved, just resyncing wallet db ;)

**Reply by luis360** [reply]
How to resync wallet db？

**Reply by mavaro** [reply]
Hi!
I did what I saw in https://github.com/Chia-Network/chia-blockchain/issues/3677#issuecomment-831964990
